app/controllers/api/jobs_controller.rb

```diff
@@ -4,7 +4,10 @@ class JobsController < Api::ApiController
     before_action :set_job, only: [:show, :retry, :update]
 
     def index
-      @jobs = Job.where(application_id: current_application.id)
+      @jobs = Job.order('created_at DESC').page(params[:page]).per(params[:per_page])
+      @jobs = @jobs.where(application_id: current_application.id)
+      @jobs = @jobs.incomplete if params[:status] == 'incomplete'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.